workspace.json

```diff
@@ -92,7 +92,9 @@
               "inspect": false
             }
           },
-          "outputs": ["{options.outputPath}"]
+          "outputs": [
```

Comment:
Why the formatting though?

Reply:
weird behaviour - slightly less dramatic changes via yarn format but still a minor change in workspace - not sure how this is happening...